checker/tests/signedness/SignednessGlbIssue.java

```diff
@@ -17,7 +17,7 @@ void m2() {
         @Unsigned int c = field;
     }
 
-    void m3() {
-        @Unsigned int c = this.field;
+    void m3(@IntRange(from = 0, to = Integer.MAX_VALUE) int array[]) {
+        @Unsigned int c=array[0];        
```

Comment:
The formatting is incorrect.

Reply:
Yes

Reply:
You don't need to respond "yes".  Just fix the problem.  Thanks.